### PR TITLE
add discourse and discord to community nav

### PR DESCRIPTION
**Pull request by thedavidprice**

### CODE REVIEW

File: code/html/_nav.html [hunk 1]
```diff
@@ -123,5 +124,8 @@
       </svg>
       <span class="ml-2">Community</span>
     </a>
+    <ul class="hidden ml-6 mt-2 mb-8">
+      @@include('community_nav')
```
Comment (thedavidprice) [inline code/html/_nav.html:128]:
```suggestion
      <a href="https://community.redwoodjs.com/``" target="_blank">Discourse Forum</a>
    <ul class="hidden ml-6 mt-2 mb-8">
      <a href="https://discord.gg/jjSYEQd" target="_blank">Discord Chat</a>
```

**Comment by thedavidprice** [comment]
@cannikin @jtoar This makes sense to add to the Community section of the nav, correct? And, if so, did I handle the code correctly?

Screen Shot 2020-07-13 at 11 33 42 PM

**Comment by cannikin** [comment]
Do you want to remove the links from the header now, or this is just in addition to?

**Comment by thedavidprice** [comment]
@cannikin I propose "in addition to" -- functionally I know it's redundant, but I think there's UX case for both.

**Pull request merged**